utils/net_utils/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "net_utils"
+version = "0.1.0"
+authors = ["ChainSafe Systems <info@chainsafe.io>"]
+edition = "2018"
+
+[dependencies]
+indicatif = "0.15.0"
```

Comment:
Can you share the same progress bar implementation as paramfetch? Doesn't matter much to me which one is used, but it's weird to see multiple types of bars

Reply:
using `pbr` now